### ISSUE TITLE
Enable users to run analyses on public data sets

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#701]
There is no easy way for users to run analyses on data sets shared with the Public group. User should be able to run analyses on those data sets the same way as if they were owned by the user (no additional steps required).

**Comment by ngehlenborg** [comment]
Related to  #249.

**Comment by jkmarx** [comment]
@ngehlenborg @hackdna Since users can import into their own space and run analyses, do we still want this feature?

**Comment by hackdna** [comment]
Yes: importing a public dataset for analysis is an extra step that I think should happen transparently to users.
